charts/api/templates/deployment-app-backend.yaml

```diff
@@ -216,15 +216,28 @@ spec:
                 name: {{ template "api.fullname" . }}-app-passport-keys
                 {{- end }}
                 key: oauth-private.key
-      {{- if .Values.app.gce.serviceAccountSecret }}
           volumeMounts:
+            {{- if .values.app.gce.serviceAccountSecret }}
             - name: "service-account-wbstack-api"
               mountPath: "/var/run/secret/cloud.google.com"
+            {{- end }}
+
+            {{- if .Values.configMap.name }}
+            - name: "mediawiki-configMap"
+              mountPath: {{ .Values.configMap.mountPath }}
```

Comment:
I think probably this could become hardcoded. I can't imagine we'll want to change it regularly particularly since where we read it from will be specified in the api image